src/DotVVM.Framework/Utils/ReflectionUtils.cs

```diff
@@ -238,14 +249,16 @@ public static Type FindType(string name, bool ignoreCase = false)
             var split = name.Split(',');
             name = split[0];
 
-            var assemblies = ReflectionUtils.GetAllAssemblies();
+            var assemblies = ReflectionUtils.GetAllAssemblies().ToList();
```

Comment:
Note that this `ToList` not very likely to improve performance, since `assemblies` are only iterated once.